src/playlist-loader.js

```diff
@@ -207,9 +207,11 @@ const PlaylistLoader = function(srcUrl, withCredentials) {
 
   loader.stopRequest = () => {
     if (request) {
-      request.onreadystatechange = null;
-      request.abort();
+      let oldRequest = request;
```

Comment:
ehh I think oldRequest works when I look at it some more. The only other thing I could think of naming it would be `req`